pdr_backend/pdr_dashboard/test/resources.py

```diff
@@ -78,3 +84,54 @@ def start_server_and_wait(dash_duo, app):
     dash_duo.wait_for_element("#predictoors_table")
     dash_duo.wait_for_element("#feeds_table tbody tr")
     dash_duo.wait_for_element("#predictoors_table tbody tr")
+
+
+def _navigate_to_feeds_page(dash_duo):
+    dash_duo.wait_for_element("#feeds")
+    dash_duo.find_element("#feeds").click()
+    time.sleep(1)
```

Comment:
instead of time.sleeps(), can we use dash duo's wait for element to be visible and pick a relevant element inside the feeds page? It's cleaner, and could save us some time. The 1s sleep is arbitrary, while a "wait for element to be visible" is more relevant here. Btw those functions do have a max timeout so they should also take care of the "max waiting time" part.

Reply:
I agree, if the action is not a generic one such as clicking a button which could have many different effects and if the action does some changes to the other elements.
In the line from above we can replace it, but in many other places we can't